plugins/repository-azure/src/test/java/org/elasticsearch/repositories/azure/AzureRepositorySettingsTests.java

```diff
@@ -103,4 +100,31 @@ public void testReadonlyWithPrimaryAndSecondaryOnlyAndReadonlyOff() throws Stora
             .put("readonly", false)
             .build()).isReadOnly(), is(false));
     }
+
+    public void testChunkSize() throws StorageException, IOException, URISyntaxException {
+        // default chunk size
+        AzureRepository azureRepository = azureRepository(Settings.EMPTY);
+        assertEquals(AzureStorageService.MAX_CHUNK_SIZE, azureRepository.chunkSize());
+        assertThat(azureRepository.chunkSize().getBytes(), greaterThanOrEqualTo(0L));
```

Comment:
I don't understand the value of this assertion given the assertion on the previous line?